R/system.R

```diff
@@ -495,6 +498,49 @@ getDataFromGoogleBigQueryTable <- function(project, dataset, table, page_size =
                  table_info = NULL, max_pages = max_page)
 }
 
+#' API to extract datafrom google BigQuery table to Google Storage
+#' @export
+extractDataFromGoogleBigQueryTableToStorage <- function(project, dataset, table, destinationUri, tokenFileId){
```

Comment:
About this function name, in other part of the code, google cloud storage is referred to as "CloudStorage", but here only as "Storage", which looks a little inconsistent.
Also, thought that "Table" of "BigQueryTable" is kind of redundant and can be omitted.